extension/src/Common.ts

```diff
@@ -67,3 +67,16 @@ export function mkDirByPathSync(targetDir: string) {
 }
 
 
+export function formatToday() {
+  var d = new Date(),
+    month = '' + (d.getMonth() + 1),
+    day = '' + d.getDate(),
+    year = d.getFullYear();
```

Comment:
I think I would prefer declaring month, day, and year as strings and using `toString()` when assigning. At the moment month and day are of type `string` whilst year is a `number`, they might as well al be strings?

Reply:
agreed!